### ISSUE TITLE
[Server] Permissions for signs?

### ISSUE THREAD

**Issue by ErdbeerbaerLP** [Team-Fruit/SignPicture#33]
I think for servers you should be able to manage permissions (ex. with ftbutilities). Because some players just post bad pictures on signs.

**Comment by Carhenia** [comment]
I agree, I've always had to ban vanilla signs so people don't create them lol

**Comment by sklll** [comment]
Definitely it's must have for using on servers. Please make sure it works ok with ForgeEssentials (if you decide to build this great feature ofc).
